Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.0'
```

Comment:
toàn bộ file ruby dùng nháy dôi nhé em, em self check file này và xóa bớt các line comment đi